packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -441,20 +520,12 @@ const VirtualListBaseFactory = (type) => {
 
 		jumpToSpottableItem = (keyCode, target) => {
 			const
-				{cbScrollTo, dataSize, isItemDisabled} = this.props,
+				{cbScrollTo, dataSize, isItemDisabled, wrap, wrapAnimated} = this.props,
```

Comment:
You could add `rtl` to get this.props.rtl instead of `rtl = this.props.rtl,`.

Reply:
👍